### ISSUE TITLE
[Accessibility] 1xMissing form label, 2xEmpty button and 1xBroken ARIA reference - Add Rental outside of the app (pop-up)

### ISSUE THREAD

**Issue by cmunteanuRW** [Talleh/VolleyballPub#14]
**Steps to reproduce:**

https://github.com/user-attachments/assets/9b69ea07-f0ce-4d7a-9763-90d617d22c47

**Actual Result:**

![Image](https://github.com/user-attachments/assets/2ec752fe-19a2-4e00-92e9-b0261a1a4974)

**Expected Result:**
No accessibility issues are present
